FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -204,12 +204,18 @@ PARAMETERs are: [ ] = optional; < > = user supplied value
             (ii) you want to be able to run some flag in recon-all, without also regenerating the surfaces.
                  e.g., [--existing-subject --extra-reconall-arg=-show-edits --no-conf2hires]
 
+  [--pipeline-mode={HCPStyleData, LegacyStyleData}]
```

Comment:
Should be `--processing-mode`

Reply:
fixed.

Reply:
This isn't fixed. Still showing as `--pipeline-mode`.

Reply:
It was in the latest push. Probably did it at the same time.